packages/tools/devtools/devtools-view/src/components/graphs/DynamicComposedChart.tsx

```diff
@@ -0,0 +1,394 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Recharts does not have types for many objects at this time. For now, these eslint disable directive should be active until we create our own types for recharts
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { Theme } from "@fluentui/react-components";
+import React, { useState } from "react";
```

Comment:
Nit, can we just call `React.useState()` below so we don't have to mix the import?

Reply:
Updated.